src/domUpdates.js

```diff
@@ -191,21 +183,11 @@ let domUpdates = {
     amount.value++;
   },
 
-  addToMyRecipes(event, currentUser, recipeData, ingredientsData) {
-    if (event.target.className === "card-apple-icon") {
-      let cardId = parseInt(event.target.closest(".recipe-card").id)
-      if (!currentUser.favoriteRecipes.includes(cardId)) {
-        event.target.src = "./images/apple-logo.png";
-        currentUser.saveRecipe(cardId);
-      } else {
-        event.target.src = "./images/apple-logo-outline.png";
-        currentUser.removeRecipe(cardId);
-      }
-    } else if (event.target.id === "exit-recipe-btn") {
-      this.exitRecipe();
-    } else if (this.isDescendant(event.target.closest(".recipe-card"), event.target)) {
-      this.openRecipeInfo(event, recipeData, ingredientsData);
-    }
+  // ************ HELPERS *************** //
+  capitalize(words) {
+    return words.split(" ").map(word => {
+      return word.charAt(0).toUpperCase() + word.slice(1);
+    }).join(" ");
```

Comment:
Much more organized now. I was curious about the 164-173 indentation earlier. Thank you for updating that